Sources/SnapshotAssertion/Diffable.swift

```diff
@@ -0,0 +1,81 @@
+import UIKit
+import XCTest
+
+extension Data: Diffable {
+  public static var diffableFileExtension: String? {
+    return nil
+  }
+
+  public var diffableData: Data {
+    return self
+  }
+
+  public func diff(comparing other: Data) -> XCTAttachment? {
+    return nil
+  }
+}
+
+extension Data: Snapshot {
+  public var snapshotFormat: Data {
+    return self
+  }
+}
+
+extension String: Diffable {
+  public static var diffableFileExtension: String? {
+    return "txt"
+  }
+
+  public var diffableData: Data {
+    return self.data(using: .utf8)!
+  }
+
+  public func diff(comparing other: Data) -> XCTAttachment? {
+    return nil
```

Comment:
should we at least just do an attachment of a string with `current: \(self)\nnew: \(other)` or something?